test/common/upstream/cluster_manager_impl_test.cc

```diff
@@ -753,6 +754,187 @@ TEST_F(ClusterManagerImplTest, ClusterProvidedLbNotConfigured) {
                             "'cluster_0' provided one. Check cluster documentation.");
 }
 
+class CustomLbFactory : public TypedLoadBalancerFactoryBase {
+public:
+  CustomLbFactory() : TypedLoadBalancerFactoryBase("envoy.load_balancers.custom_lb") {}
+
+  ThreadAwareLoadBalancerPtr
+  create(const PrioritySet&, ClusterStats&, Stats::Scope&, Runtime::Loader&,
+         Random::RandomGenerator&,
+         const ::envoy::config::cluster::v3::LoadBalancingPolicy::Policy&) override {
+    return std::make_unique<ThreadAwareLbImpl>();
+  }
+
+private:
+  struct LbImpl : public LoadBalancer {
```

Comment:
```suggestion
  class LbImpl : public LoadBalancer {
  public:
```

per style guide, same for below

Reply:
Done, thanks.